### PR TITLE
Mention command features. Fixes #502

### LINKED ISSUE #502: Show off Hovertext and tab-completion
There are some nifty features provided by the Sponge implementations (SV and SF) that make things easy for users and admins. For example, the `/sponge plugins` command provides a list of plugins in-game that will reveal their version when Hovering over the name.
Similarly, tab auto-completion works reasonably well with `/sponge` commands, in-game and via console. 
I didn't see any reference to these ease-of-use features in the docs, and I think they're worth mentioning. This would apply to the 3.1.0 docs branch as well as current.

### PR DESCRIPTION
**Pull request by ST-DDT**
Fixes #502 

Does this suffice to show of the command features? I'm not sure on the positioning and the image either.
If anyone has a suggestion for either then please tell me.

### CODE REVIEW

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/fd9996d/.

Here are some links to the pages that were modified:

- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/fd9996d/server/spongineer/commands.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/733#issuecomment-410514226), not to the direct url to the preview._

File: source/server/spongineer/commands.rst [hunk 1]
```diff
@@ -222,3 +222,25 @@ Command               Description                               Permission
...
 for vanilla Minecraft commands on a Sponge server are of the structure ``minecraft.command.<command>``.
+
+Command Features
+================
+
+.. image:: /images/command-features.png
+    :align: center
+    :alt: command features
+
+Sponge and most Sponge plugins support additional command features such as auto completion or hoverable text. The image
```
Comment (Inscrutable) [inline source/server/spongineer/commands.rst]: auto completion *and* hoverable text. (we have both)

File: source/server/spongineer/commands.rst [hunk 2]
```diff
@@ -222,3 +222,25 @@ Command               Description                               Permission
...
+above shows this using the ``/sponge plugins`` command (yellow box). The elements in that list can be hovered to get
```
Comment (Inscrutable) [inline source/server/spongineer/commands.rst]: "... shows the *output* of the `/sponge plugins`` command." (the other bits show off the features).

File: source/server/spongineer/commands.rst [hunk 3]
```diff
@@ -222,3 +222,25 @@ Command               Description                               Permission
...
+addition information such as the current version number (red box). Some elements in the shown text have additional
```
Comment (Inscrutable) [inline source/server/spongineer/commands.rst]: "Some elements in the *example above* have ..."

File: source/server/spongineer/commands.rst [hunk 4]
```diff
@@ -222,3 +222,25 @@ Command               Description                               Permission
...
+actions bound to them. For example the plugin entries in that list can be clicked to show more detailed information
+(purple box) about that plugin. This is equivalent to sending the ``/sponge plugins <pluginId>`` command. The auto
+completion can be triggered by pressing tab. Entering ``/sponge plugins ﻿`` (with a trailing space) and then pressing
+tab will show a list of possible values (turquoise box) that can be used in that context. Pressing tab again will
+iterate the possibilities, if there are more than one. Theoretically it is possible to write most commands by pressing
```
Comment (Inscrutable) [inline source/server/spongineer/commands.rst]: "if any." is shorter than "if there are more than one".

File: source/server/spongineer/commands.rst (hunk 2 above)
Comment (Inscrutable) [inline source/server/spongineer/commands.rst]: "Elements of this list may be hovered *over* to get ..."

**Review by Inscrutable — COMMENTED** [review]
Very good indeed. I'm not sure if the image looks better at the top or the bottom. Putting it below might suit our style more, but it'll necessitate some changes in the text too. Further opinions sought.

**Comment by ST-DDT** [comment]
Thanks for your review. I moved the image to the bottom as suggested (I don't prefer any variant more than the other).

**Pull request merged**